translations/base-en.yaml

```diff
@@ -266,6 +266,10 @@ dialogs:
         title: New Marker
         desc: Give it a meaningful name, you can also include a <strong>short key</strong> of a shape (Which you can generate <a href="https://viewer.shapez.io" target="_blank">here</a>)
 
+    editMarker:
+        title: Edit Marker
+        desc: Give it a new meaningful name, you can also include a <strong>short key</strong> of a shape (Which you can generate <a href="https://viewer.shapez.io" target="_blank">here</a>)
```

Comment:
Why duplicate the translation key here? :) You can just use the same one

Reply:
https://github.com/tobspr/shapez.io/blob/2722366d686fd551a8edceebafc8e78c1e40e976/translations/base-en.yaml#L283-L286
Prepared another title using the same translation key